### DISCUSSION TITLE
v1.6 - does it still need a rebuild to work with Houdini?

### DISCUSSION THREAD

**Discussion by mpjohn** [dreamworksanimation/openmoonray#160] [General]
Hi There,
Happy to see a new build of Moonray. I'm hoping that this version will be compatible with some version of Houdini without the need to rebuild it or hack it somehow. Is that the case? And if so does anyone know which version of Houdini?

Thanks
M

**Comment by randypacker** [comment]
Hi mpjohn,

Yes, you would need to (re)build v1.6 with the [SideFx USD](https://github.com/sideeffects/USD) version to work in Houdini.   We have tested the SideFx-compatible versions of USD-0.21.08 (for Houdini 19.0) and 0.22.05 (for Houdini 19.5) successfully with hdMoonray.   Let us know how it goes, cheers!

**Reply by mpjohn** [reply]
Hi Randy,

Would it be possible to add build instructions for Houdini?  Build instructions that don't work with a DCC are of very limited value to those of us who want to try to actually use the renderer ( outside of people in Dreamworks ).
I know there are a few threads about people trying to build against Houdini but in my opinion this needs to be tested and documented properly if it's to be of any real utility.

M
